docs/plugins/discogs.rst

```diff
@@ -76,6 +76,12 @@ whereas with ``index_tracks`` disabled you'd get::
 
 This option is useful when importing classical music.
 
+Other configurations available under ``discogs:`` are:
+
+- **append_style_genre**: Appends the style (if found) to the genre tag, useful if you would like more granular genre styles added to music file tags
```

Comment:
Maybe including an example here would be useful? Your "Electronic; Techno" example is pretty illustrative, for instance. You could say that, by default, you just get "Electronic" but get the longer value when the option is enabled.